smf/clickhouse_keeper/method_script.sh

```diff
@@ -10,4 +10,4 @@ set -o pipefail
 # Otherwise, it does not automatically detect the configuration files, nor does
 # it append them when necessary
 cd /opt/oxide/clickhouse_keeper/
```

Comment:
Are the `cd` and the comment explaining it no longer needed?

Reply:
Yup. Done in [3a6b659](https://github.com/oxidecomputer/omicron/pull/6894/commits/3a6b6592f1a8a02816ab1ac7495b98e56164b9b4)